### DISCUSSION TITLE
Why i can't get proper sequence when i import JSON in Scalar desktop app

### DISCUSSION THREAD

**Discussion by aditya05062002** [scalar/scalar#4076] [Q&A]
Can some one help me why i get different sequence from original in scalar desktop app. In Scalar Document i get proper sequence like post, get, put, delete, search but in scalar desktop app i get sequence  like delete, get, post, put, get why that happening if i import same JSON into both of them 
![Screenshot (81)](https://github.com/user-attachments/assets/780da2c4-6b40-4250-ba7a-a9064b9c9c59)
![Screenshot (83)](https://github.com/user-attachments/assets/abb48198-3b06-44db-bc4a-5298f02252e6)

**Comment by marclave** [comment]
hey @aditya05062002 this is a bug! lets fix it :)

**Comment by antlio** [comment]
great catch @aditya05062002 thanks for the feedback, a PR has been opened here:
https://github.com/scalar/scalar/pull/4092

**Reply by aditya05062002** [reply]
Thanks for the feedback!
